app/lib/discourse_automation/triggers/user_added_to_group.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+DiscourseAutomation::Triggerable::USER_ADDED_TO_GROUP = 'user_added_to_group'
+
+DiscourseAutomation::Triggerable.add(DiscourseAutomation::Triggerable::USER_ADDED_TO_GROUP) do
+  on_update do |automation, metadata|
```

Comment:
Does this one do anything?

Reply:
I'm torn on this ATM... basically this hook let's you define some logic when you update an automation with this trigger: removes scheduled things, update some fields... and i'm not sure yet for this case. Will remove the block for now.